### PR TITLE
Added wash trade status

### PR DESCRIPTION
**Pull request by peterbarrow**
Added a new table to show the order status after a wash trade has been identified.

### CODE REVIEW

**Review by jeremyletang — APPROVED** [review]

File: specs/0024-order-status.md [hunk 1]
```diff
@@ -57,3 +57,11 @@ Note: The last row in the table below is added for clarity rather than being a l
 |      GTT      | Partial |    No   |         No        |        Yes        |      Stopped     |
 |      GTT      |   Yes   |    No   |         No        |         No        |      Filled      |
 |      GTT      |   Yes   |   Yes   |         No        |         No        |      Filled      |
+
+## Wash trading
+If an order would be filled or partially filled with an existing order from the same traderID, the order is rejected. Any existing fills that happen before the wash trade is identified will be kept.
+
```
Comment (tamlyn10) [inline specs/0024-order-status.md:63]: @peterbarrow can you explain how the existing fills before the wash trade is identified happens? Also, @barnabee somewhere we did spec something about wash trading - do you recall where? I'm can't find it now. I thought it was in the checking of the order spec 🤔

Comment (tamlyn10) [reply specs/0024-order-status.md:63]: Actually @peterbarrow I understand what you're saying now... sorry,  a bit under the weather and dopey today!

**Review by tamlyn10 — APPROVED** [review]

File: specs/0024-order-status.md (hunk 1 above)
Comment (barnabee) [reply specs/0024-order-status.md:63]: @tamlyn10 wash trades are discussed in order submission spec: https://github.com/vegaprotocol/product/blob/master/specs/0025-order-submission.md

Comment (tamlyn10) [reply specs/0024-order-status.md:63]: Great, somehow that spec escaped me. OK @peterbarrow I think this spec is ready to merge..

**Pull request merged**